src/local/delete.go

```diff
@@ -69,32 +69,35 @@ func (m *Manager) DeleteEmptyDirectories(trackedPaths []string) error {
 			return nil
 		}
 
-		// Skip ignored
-		if utils.IsIgnoredDir(path, d) {
-			return filepath.SkipDir
-		}
+		// Stop on ignored dir
+		isIgnoredDir := utils.IsIgnoredDir(path, d)
 
 		// Found a directory -> store path
-		if d.IsDir() {
+		if !isIgnoredDir && d.IsDir() {
 			emptyDirs.Set(path+string(os.PathSeparator), true)
 			return nil
 		}
 
-		// Is file -> all parent dirs are not empty
+		// Found file/ignored dir -> all parent dirs are not empty
```

Comment:
Vyssie uz nie je return, takze toto plati aj pre `isIgnoredDir = true`.
Ak najdeme ignorovany adresar, .... tak to znamena, ze nadradeny adresar NIE je prazdny.